cypress/integration/userCanUseRatingFunctionality.feature.js

```diff
@@ -0,0 +1,62 @@
+describe('User can use rating functionality', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://fakest-newzz.herokuapp.com/api/articles', {
+      fixture: 'articles.json',
+    });
+    cy.intercept('GET', 'https://fakest-newzz.herokuapp.com/api/articles/3', {
+      fixture: 'specificArticle.json',
+    });
+    cy.visit('/');
+  });
+  describe('Successfully', () => {
+    beforeEach(() => {
+      cy.intercept('PUT', 'https://fakest-newzz.herokuapp.com/api/ratings', {
+        statusCode: 200,
+      });
+    });
+
+    it('is expected to display average rating on each article card', () => {
+      cy.get('[data-cy=articles-container]').within(() => {
+        cy.get('[data-cy=article-card-0]').within(() => {
+          cy.get('[data-cy=rating]')
+            .find('i')
+            .eq(3)
+            .should('have.attr', 'aria-checked', 'true');
+        });
+      });
+    });
+    it('is expected to display average rating on breaking news', () => {
+      cy.get('[data-cy=breaking-rating]')
+        .find('i')
+        .eq(4)
```

Comment:
So you don't click this stuff, and .eq will make value of rating to be equal 4 in that case?

Reply:
`.eq(4)` takes the fourth element in the array

Reply:
As in clicks the 5 star :)

Reply:
cool